identity-account/Cargo.toml

```diff
@@ -48,6 +48,7 @@ optional = true
 rand = { version = "0.8" }
 rusty-fork = { version = "0.3" }
 tokio = { version = "1.14", features = ["macros", "rt", "rt-multi-thread", "sync"] }
+identity-account= { path = ".", features = ["send-sync-storage"] }
```

Comment:
As discussed, I think `send-sync-storage` should be a default feature on `identity-account` instead.